Source/OrchBase/Composer.h

```diff
@@ -48,6 +49,17 @@ typedef struct devTypStrings_t
     std::string varsHCommon;
 } devTypStrings_t;
 
+typedef enum softswitchLogHandler_t
+{
+    disabled = 0,
+    trivial
+} ssLogHandler_t;
+
+typedef enum softswitchLoopMode_t
+{
+    standard = 0,
+    priInstr
+} ssLoopMode_t;
 

```

Comment:
These look a lot like fancy booleans to me, which is fine if you're planning to add more entries.

Reply:
At the moment, they are. Intention is to expand - we already have a "new" log handler part done, and we want to be able to prioritise sending. I have just not had time to implement them yet, but design is there to support.